src/lib.rs

```diff
@@ -671,6 +674,31 @@ impl Navigator {
         self.led.set_led_all(state)
     }
 
+    /// Set the values of the neopixel LED array.
+    ///
+    /// # Arguments
+    ///
+    /// * `array` - A 2D array containing RGB values for each LED.
+    /// Each inner array is a [u8; 3] representing the Red, Green and Blue from a LED.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use navigator_rs::{Navigator};
+    ///
+    /// let mut nav = Navigator::new();
+    ///
+    /// nav.init();
+    /// let mut leds = [[0, 0, 255], [0, 255, 0], [255, 0, 0]];
```

Comment:
`let mut leds = [[255, 255, 255];`

the other indexes will not work, this example only turn on the blue LED.

Reply:
@patrickelectric

Reply:
The idea is to support LED strips. Right now the function only sets our board LED. Since the only LED that exists is the first, it'll only show blue.